lib/srv/exec_test.go

```diff
@@ -515,3 +515,6 @@ func (a *fakeLog) WaitForDelivery(context.Context) error {
 func (a *fakeLog) Close() error {
 	return trace.NotFound("")
 }
+func (a *fakeLog) EmitAuditEvent(ctx context.Context, e events.AuditEvent) error {
```

Comment:
Hmm, I don't think fakeServer is even using auditLog - tests pass even if it does not have it so I'd just remove it.

Reply:
The tests do not pass if it has it does not have it